packages/atlas/src/views/playground/Playgrounds/NftPurchase.tsx

```diff
@@ -1,15 +1,20 @@
-import React from 'react'
+import React, { useState } from 'react'
 
 import { Button } from '@/components/_buttons/Button'
+import { TextField } from '@/components/_inputs/TextField'
 import { useNftActions } from '@/providers/nftActions'
-import { NftPurchaseView } from '@/views/viewer/NftPurchaseView'
 
 export const NftPurchase: React.FC = () => {
   const { openNftPurchase } = useNftActions()
+
+  const [nftId, setNftId] = useState('')
+
   return (
     <>
-      <Button onClick={() => openNftPurchase('1')}>Open auction overlay</Button>
-      <NftPurchaseView />
+      <TextField value={nftId} onChange={(e) => setNftId(e.currentTarget.value)} label="Video ID" />
+      <br />
```

Comment:
Why do we need `<br />` tag?

Reply:
I don't know I wanted to add some space here - this is only a playground. But okay, I'll remove this

Reply:
Oh, sorry I was so outraged by this that I didn't notice it was a playground 😛